src/app/modules/design-patterns/enums/colors.ts

```diff
@@ -0,0 +1,5 @@
+export enum COLOR {
```

Comment:
change to `src/app/modules/design-patterns/observer/types/colors.ts`

Reply:
should we need this file?
we aren't using it anywhere

Reply:
no, remove it